From 6db066205ede0e06e7c4f38072a15229a741e422 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Wed, 1 Apr 2020 21:44:26 +0530
Subject: [PATCH] greybus: Remove Android.mk

Remove Android.mk as redefine errors are seen when multiple
kernels projects are used in the Android build.

Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 drivers/staging/greybus/tools/Android.mk | 10 ----------
 1 file changed, 10 deletions(-)
 delete mode 100644 drivers/staging/greybus/tools/Android.mk

diff --git a/drivers/staging/greybus/tools/Android.mk b/drivers/staging/greybus/tools/Android.mk
deleted file mode 100644
index fdadbf611757..000000000000
--- a/drivers/staging/greybus/tools/Android.mk
+++ /dev/null
@@ -1,10 +0,0 @@
-LOCAL_PATH:= $(call my-dir)
-
-include $(CLEAR_VARS)
-
-LOCAL_SRC_FILES:= loopback_test.c
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE := gb_loopback_test
-
-include $(BUILD_EXECUTABLE)
-
-- 
2.39.2

